common/static/common/js/discussion/utils.js

```diff
@@ -263,9 +278,9 @@
 
         DiscussionUtil.formErrorHandler = function(errorsField) {
             return function(xhr, textStatus, error) {
-                var makeErrorElem, response, _i, _len, _ref, _results;
+                var makeErrorElem, response, _i, _len, _ref, _results, $errorItem;
                 makeErrorElem = function(message) {
-                    return $("<li>").addClass("post-error").html(message);
+                    return $("<li>").addClass("post-error").html(edx.HtmlUtils.ensureHtml(message).toString());
```

Comment:
I think you could avoid the `'toString()` by using (untested):
```
return edx.HtmlUtils.setHtml(
    $("<li>").addClass("post-error"),
    message
);
```
I think we'd prefer using HtmlUtils calls that have `ensureHtml` built in where they exist (like `setHtml`), and save the above style (which is accurate) for JQuery functions that aren't supported directly in HtmlUtils.